viewer/components/util/util.js

```diff
@@ -0,0 +1,244 @@
+(function() {
+
+  'use strict';
+
+  /**
+   * Util Directives
+   * Exposes utility directives
+   */
+  angular.module('moloch.util', [])
+
+    /**
+     * Protocol filter
+     * Displays the protocol string
+     *
+     * @example
+     * '{{session.pr | protocol}}'
+     */
+    .filter('protocol', () => {
+      return (input) => {
+        var result;
+
+        switch (input) {
```

Comment:
style nit, but only need return "string", don't need result = "string"; return result.  Code copied had it because it added stuff.  Could even do it with an object lookup.